docs/components/rendering_iterables.md

```diff
@@ -50,26 +110,25 @@ class IterIndexState(rx.State):
         "red",
         "green",
         "blue",
-        "yellow",
-        "orange",
-        "purple",
     ]
 
 
 def enumerate_foreach():
-    return rx.grid(
+    return rx.vstack(
         rx.foreach(
             IterIndexState.color,
-            lambda color, index: rx.box(rx.text(index), bg=color)
+            lambda color, index: rx.box(
+                rx.button(f"{index + 1}. {color}"),
+                padding_y="0.5em",
+            ),
         ),
-        columns="6",
     )
 
 ```
 
-## Dictionary
+## Iterating Dictionaries
```

Comment:
might be nice to mention the necessity of typing correctly here so that foreach works